prov/efa/src/efa_errno.h

```diff
@@ -58,16 +58,17 @@
 	_(1,	FLUSHED,			Flushed during queue pair destroy)		\
 	_(2,	LOCAL_ERROR_QP_INTERNAL_ERROR,	Internal queue pair error)			\
 	_(3,	LOCAL_ERROR_INVALID_OP_TYPE,	Invalid operation type)				\
-	_(4,	LOCAL_ERROR_INVALID_AH,		Invalid address handle)				\
+	_(4,	LOCAL_ERROR_INVALID_AH,		Invalid address handle (local))			\
 	_(5,	LOCAL_ERROR_INVALID_LKEY,	Invalid local key (LKEY))			\
 	_(6,	LOCAL_ERROR_BAD_LENGTH,		Message too long)				\
-	_(7,	REMOTE_ERROR_BAD_ADDRESS,	Destination ENI is down or does not run EFA)	\
```

Comment:
I am not sure we can change this checking the rdma-core symbols.

before the rdma-core err code split,   `REMOTE_ERROR_BAD_ADDRESS` still means either bad MR or bad AH?

Reply:
The PR can wait for the RDMA core change to be merged. This would be simpler if RDMA core had its own flavor of `strerror()`; we wouldn't need to worry about which version of RDMA core is reporting a given errno.